client/src/components/Monument/Monument.js

```diff
@@ -35,13 +35,22 @@ export default class Monument extends React.Component {
             this.callEndpoint(`/api/tags/?monumentId=${monumentId}`)
         ]).catch(err => error = err);
 
+        images.splice(0);
+
         if (error) {
             console.error(error);
             this.setState({error: error});
             return;
         }
 
-        this.setState({monument, contributions, images, references, tags});
+        this.setState({monument, contributions, images: [
```

Comment:
This is good for now, but we'll definitely want to remove this once we get images stored in the back-end.

Reply:
also derp. although it's handy to have so I'll leave it in and put a todo